images/linux/scripts/installers/cmake.sh

```diff
@@ -9,7 +9,8 @@ echo "Checking to see if the installer script has already been run"
 if command -v cmake; then
     echo "cmake is already installed"
 else
-	curl -sL https://cmake.org/files/v3.17/cmake-3.17.0-Linux-x86_64.sh -o cmakeinstall.sh \
+	VERSION=$(curl -s https://api.github.com/repos/Kitware/CMake/releases/latest | grep "tag_name" | cut -d'v' -f2 | cut -d'"' -f1)
```

Comment:
I suggest to use `jq` instead of grep here. It is more reliable way: `jq -r '.tag_name | ltrimstr("v")'`
```bash
VERSION=$(curl -s https://api.github.com/repos/Kitware/CMake/releases/latest | jq -r '.tag_name | ltrimstr("v")')
```

Reply:
Good suggestion, thanks!